EnglishTranslation-master/QUEST.tsv

```diff
@@ -1074,30 +1074,30 @@ QUEST_20150317_001073	198 [Siauliai46]
+QUEST_20150317_001080	This is an old story of Fedimian before the Goddess Statue was built.
+QUEST_20150317_001081	A long time ago, the people of Fedimian were quite normal, {nl}not much different from the ordinary people of the rest of the world. {nl}However, the people of Fedimian did differ from their worldly peers in one aspect.
 QUEST_20150317_001082	The people of Fedimian were very cautious.{nl}In the sense that they take the word 'cautious' much more seriously. {nl}These people were as cautious around fire as they were without.
 QUEST_20150317_001083	The Fedimians were modest about this with others. {nl}Even those who stayed at Fedimian for a while could not pick up the trait.
 QUEST_20150317_001084	Thanks to this behavior, the people of Fedimian fared better during tough times, {nl}as they were able to sense danger coming.
-QUEST_20150317_001085	Because of this, no few words are said about the cautiousness of the Fedimian people.
-QUEST_20150317_001086	In Fedimian, if someone dropped money on the ground, it will be gone forever. {nl}In a world where money is mostly consist of silver coins, {nl}only the creditors would know exactly how much money has been lost in this manner.
-QUEST_20150317_001087	Why money disappears into the ground at Fedimian remains unknown. {nl}Some people believe the Goddess Zemyna absorbs money that makes contact with her area. {nl}But most commoners believe this was done to reduce the demand of silver coins from creditors.
-QUEST_20150317_001088	By believing that the goddess herself takes the silver,{nl}the people found it fruitless trying to dig up silver on her grounds. {nl}The fact that you can offer silver coin at Fedimian as tribute to her reinforces this idea.
-QUEST_20150317_001089	In any case, the fact that one could lose money by dropping it on Fedimian soil,{nl}has made Fedimians very careful in handling their money.
+QUEST_20150317_001085	Because of this, no few words were said about the cautiousness of the Fedimian people.
+QUEST_20150317_001086	In Fedimian, if someone dropped money on the ground, it would be gone forever. {nl}In a world where money mostly consists of silver coins, {nl}only the creditors would know exactly how much money has been lost in this manner.
+QUEST_20150317_001087	Why money disappears into the ground at Fedimian remains unknown. {nl}Some people believe the Goddess Zemyna absorbs any money that makes contact with her area. {nl}Most commoners, however, believe this was done to reduce the demand of silver coins from creditors.
+QUEST_20150317_001088	By believing that the Goddess herself takes the silver,{nl}the people found it pointless to try and dig up the silver in her grounds. {nl}The fact that you can offer silver coins at Fedimian as tribute to her reinforces this idea.
```

Comment:
Goddess -> goddess
She is already addressed as Goddess Zemyna in the previous sentence. Since this is a follow up, it's logical to think that this is about that specific goddess.